care/emr/signals/patient_identifier/cache_invalidation.py

```
@@ -0,0 +1,24 @@
+from django.core.cache import cache
+from django.db.models.signals import post_save
+
+from care.emr.models.patient import PatientIdentifierConfig
+from care.emr.resources.base import model_cache_key, model_string
+from care.facility.models import Facility
+
+
+def invalidate_facility_cache_on_identifier_change(sender, instance, **kwargs):
```

Ruff (0.14.14)

[warning] Unused function argument: `sender`

(ARG001)